src/lightning/fabric/fabric.py

```diff
@@ -608,6 +608,22 @@ def load(
         return self._strategy.load_checkpoint(path=path, state=state)
 
     def launch(self, function: Optional[Callable[["Fabric"], Any]] = None, *args: Any, **kwargs: Any) -> Any:
+        """Launch and initialize all the processes needed for distributed execution.
+
+        Args:
+            function: Optional function to launch when using a spawn/fork-baded strategy, for example, when using the
+                XLA strategy (``accelerator="tpu"``). The function must accept at least one argument, to which
+                the Fabric object itself will be passed.
+            *args: Optional positional arguments to be passed to the function.
+            **args: Optional keyword arguments to be passed to the function.
+
+        Returns:
```

Comment:
Can you add a note mentioning that calling launch more than once is a no-op?

Reply:
done